clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/TransferArraysV1QueryInput.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+import java.net.URI
+import java.time.OffsetDateTime
+import java.util.UUID
+
+import org.broadinstitute.clio.util.model.{
+  DocumentStatus,
+  Location,
+  RegulatoryDesignation
+}
+
+case class TransferArraysV1QueryInput(
+  documentStatus: Option[DocumentStatus] = None,
+  location: Option[Location] = None,
+  project: Option[String] = None,
+  sampleAlias: Option[String] = None,
+  version: Option[Int] = None,
+  pipelineVersion: Option[Symbol] = None,
+  workflowStartDate: Option[OffsetDateTime] = None,
+  workflowEndDate: Option[OffsetDateTime] = None,
+  cramMd5: Option[Symbol] = None,
+  cramSize: Option[Long] = None,
+  cramPath: Option[URI] = None,
+  cromwellId: Option[UUID] = None,
+  readgroupMd5: Option[Symbol] = None,
+  workspaceName: Option[String] = None,
+  regulatoryDesignation: Option[RegulatoryDesignation] = None,
```

Comment:
I don't think we use regulatoryDesignation in arrays

Reply:
We don't think we'll ever do clinical arrays? Or are you saying that we don't have that functionality now, so why add it now?